index.html

```diff
@@ -23,26 +18,39 @@
                 <div class = "small-10 large-8 small-centered column">
                     <h1 id = "web-title">Ñusic</h1>
                 </div>
-                
             </nav>
         </div>
 
         <div class="row">
             <div class="small-10 medium-10 large-8 small-centered columns">
-                <div class="search-box align-center">
-                    Search Box
-                    <input class = "text-box" type="text">
+                <div class="box align-center">
+                    Search for your Band
+                    <input class = "text-box" id = "search-box" type="text">
+                    <button type = "submit" id = "search-button">Search!</button>
+                </div>
+            </div>
+        </div>
+    
+
+        <!--Results: Playlist-->
+        <div class="row">
+            <div class="small-10 large-8 small-centered columns">
+                
+                <div class="box" id ="playlist">
```

Comment:
Switch the playlist and box down there since the playlist will pop up under the selected band concert.